backend/src/appointment/controller/mailer.py

```diff
@@ -258,11 +265,14 @@ def text(self):
 
 class ConfirmationMail(BaseBookingMail):
     def __init__(self, confirm_url, deny_url, name, email, date, duration, schedule_name, *args, **kwargs):
-        """init Mailer with confirmation specific defaults"""
+        """Init Mailer with action-required:confirm/deny specific defaults
+           To: Event owner
+        """
         self.confirmUrl = confirm_url
         self.denyUrl = deny_url
         self.schedule_name = schedule_name
-        default_kwargs = {'subject': l10n('confirm-mail-subject', {'name': name})}
+        lang = kwargs['lang'] if 'lang' in kwargs else None
```

Comment:
Do we need to do this? kwargs is passed through to the constructor (and BaseBookingMail's constructor.) and it doesn't seem like it's explicitly used in the constructor elsewhere.

Reply:
I thought it would be a good idea to not require lang to be in the kwargs, but yeah, I see it makes sense that it's there. I will update it.